Sofa/Component/LinearSolver/Direct/src/sofa/component/linearsolver/direct/BTDLinearSolver.h

```diff
@@ -40,7 +41,7 @@ namespace sofa::component::linearsolver::direct
 /// http://www.cfd-online.com/Wiki/Tridiagonal_matrix_algorithm_-_TDMA_(Thomas_algorithm)
 /// http://www4.ncsu.edu/eos/users/w/white/www/white/ma580/chap2.5.PDF
 template<class Matrix, class Vector>
-class BTDLinearSolver : public sofa::component::linearsolver::MatrixLinearSolver<Matrix,Vector>
+class BTDLinearSolver : public sofa::component::linearsolver::MatrixLinearSolver<Matrix,Vector>, public sofa::core::behavior::PartialLinearSolver
 {
 public:
     SOFA_CLASS(SOFA_TEMPLATE2(BTDLinearSolver, Matrix, Vector), SOFA_TEMPLATE2(sofa::component::linearsolver::MatrixLinearSolver, Matrix, Vector));
```

Comment:
I think it should now also declare `PartialLinearSolver` as a parent:
`SOFA_CLASS2(BTD, MatrixLinearSolver, PartialLinearSolver)`

Reply:
Well spotted